ENIGMAsystem/SHELL/Platforms/SDL/Event.cpp

```diff
@@ -253,7 +253,7 @@ void SDL_Event_Handler::mouseButtonDown(const SDL_Event *event) {
 void SDL_Event_Handler::mouseButtonUp(const SDL_Event *event) {
   int btn = SDL_map_button_enum(event->button.button);
   if (btn < 0) return;
-  SDL_CaptureMouse(SDL_FALSE);
+  if (!SDL_GetMouseState(NULL, NULL)) SDL_CaptureMouse(SDL_FALSE);
```

Comment:
Well, at least I can say by looking at the source, this does seem to be cached, so not really that much different from wParam I guess.
https://github.com/spurious/SDL-mirror/blob/3384f7b2845eeeae887386d0879a24d066678775/src/events/SDL_mouse.c#L172
https://github.com/spurious/SDL-mirror/blob/3384f7b2845eeeae887386d0879a24d066678775/src/events/SDL_mouse.c#L702